Flask/mongo_connect/mongo_connect.py

```diff
@@ -179,5 +179,5 @@ def add_syllables(language):
     return "The syllable {} has been added to the {} syllable database".format(ipa_syllable, language)
 
 if __name__ == '__main__':
-    #app.run(debug=True, host='127.0.0.1', port=5001)
-    app.run(debug=True, host='0.0.0.0')
+    app.run(debug=True, host='127.0.0.1', port=5001)
+    # app.run(debug=True, host='0.0.0.0')
```

Comment:
Don't push changes like this to master, things like this are why stuff breaks unexpectedly.
Please revert (comment one line, uncomment the other)

Reply:
+1